deployments/helm/cvmfs-csi/values.yaml

```diff
@@ -5,25 +5,25 @@
 # These can be used e.g. when defining CVMFS client configuration.
 # ConfigMap data supports go-template expressions.
 extraConfigMaps:
-# /etc/cvmfs/default.local
-- name: cvmfs-csi-default-local
-  data:
-    default.local: |
-      CVMFS_USE_GEOAPI=yes
-      CVMFS_HTTP_PROXY="http://ca-proxy.cern.ch:3128"
-
-      # It is advised to change these configs in the cache section of the helm values
-      # and leave them unchanged here, so they auto-generate.
-      CVMFS_QUOTA_LIMIT={{ .Values.cache.local.cvmfsQuotaLimit }}
-      CVMFS_CACHE_BASE=/cvmfs-localcache
-
-      {{- if .Values.cache.alien.enabled }}
-      CVMFS_ALIEN_CACHE=/cvmfs-aliencache
-      # When alien cache is used, CVMFS does not control the size of the cache.
-      CVMFS_QUOTA_LIMIT=-1
-      # Whether repositories should share a cache directory or each have their own.
-      CVMFS_SHARED_CACHE=no
-      {{- end -}}
+  # /etc/cvmfs/default.local
+  cvmfs-csi-default-local:
+    data:
```

Comment:
I think we can make the dict flatter by removing `data`.

Reply:
@gman0 Have flattened it further, please take a look.